aiohttp/connector.py

```diff
@@ -850,6 +850,39 @@ async def _resolve_host(
                     await trace.send_dns_cache_hit(host)
             return result
 
+        #
+        # If multiple connectors are resolving the same host, we wait
+        # for the first one to resolve and then use the result for all of them.
+        # We use a throttle event to ensure that we only resolve the host once
+        # and then use the result for all the waiters.
+        #
+        # In this case we need to create a task to ensure that we can shield
+        # the task from cancellation as cancelling this lookup should not cancel
+        # the underlying lookup or else the cancel event will get broadcast to
+        # all the waiters across all connections.
+        #
+        host_resolved = asyncio.create_task(
```

Comment:
Maybe, call it a `resolved_host_task`?

Reply:
I moved the code from below and didn't think about renaming it.  Sounds good to me.